src/client/pydaos/raw/daos_api.py

```diff
@@ -1374,12 +1375,18 @@ def __init__(self, context, cuuid=None, poh=None, coh=None):
         self.coh = ctypes.c_uint64(0)
         self.poh = ctypes.c_uint64(0)
         self.info = daos_cref.ContInfo()
+        self.cont_prop = None
+        self.cont_type = None
+        self.flag = None
 
     def get_uuid_str(self):
         """Return C representation of Python string."""
         return conversion.c_uuid_to_str(self.uuid)
 
-    def create(self, poh, con_uuid=None, cb_func=None):
+    def create(self, poh, con_uuid=None, layer_type=None,
```

Comment:
(pylint-too-many-branches) Too many branches (26/25)

Reply:
Is this problem critical? Not sure.